cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CaRenewalTest.java

```diff
@@ -199,6 +219,16 @@ protected CertAndKey generateSignedCert(Subject subject,
                         "new-password" + index
                 );
             }
+
+            @Override
+            protected boolean isCaCertGenerationChanged() {
+                return false;
+            }
+
+            @Override
+            protected String caCertGenerationAnnotation() {
+                return null;
+            }
```

Comment:
Aside: I wish there were a better alternative to using anonymous subclasses of Ca for this test. It makes it harder to see how the tests are different. Could you see if there's scope for collecting some of them together into named subclasses at least, rather than having a different subclass per test?

Reply:
Ok. Just done. I factored out a `MockedCa` inner class that should work as you suggested.